### ISSUE TITLE
Error building app with a timepicker

### ISSUE THREAD

**Issue by fredericoregateiro** [AvaloniaUI/Avalonia#17484]
### Describe the bug

I'm not able to use a Timepicker, adding a TimePicker to any view results on a designer exception, the build is ok but the designer stop working and i get this in the output:

```
17:17:20.341 [Verbose]  Finished AvaloniaDesigner.StartProcessAsync()
17:17:20.343 [Debug] 38060 <= "Initializing application in design mode"
17:17:20.349 [Debug] 38060 <= "Obtaining AppBuilder instance from AvaloniaApplicationTimePicker.Desktop.Program"
17:17:20.489 [Error] 38060 <= "Unhandled exception. System.TypeLoadException: Method 'get_TypographicFamilyName' in type 'Avalonia.Skia.GlyphTypefaceImpl' from assembly 'Avalonia.Skia, Version=11.1.0.0, Culture=neutral, PublicKeyToken=c8d484a7012f9a8b' does not have an implementation."
17:17:20.490 [Error] 38060 <= "   at Avalonia.Skia.FontManagerImpl.TryCreateGlyphTypeface(Stream stream, FontSimulations fontSimulations, IGlyphTypeface& glyphTypeface)"
17:17:20.490 [Error] 38060 <= "   at Avalonia.Media.Fonts.EmbeddedFontCollection.Initialize(IFontManagerImpl fontManager)"
17:17:20.491 [Error] 38060 <= "   at Avalonia.Media.FontManager.AddFontCollection(IFontCollection fontCollection)"
17:17:20.491 [Error] 38060 <= "   at Avalonia.AppBuilderExtension.<>c.<WithInterFont>b__0_0(FontManager fontManager)"
17:17:20.492 [Error] 38060 <= "   at Avalonia.AppBuilder.<>c__DisplayClass71_0.<ConfigureFonts>b__0(AppBuilder appBuilder)"
17:17:20.492 [Error] 38060 <= "   at Avalonia.AppBuilder.SetupUnsafe()"
17:17:20.493 [Error] 38060 <= "   at Avalonia.AppBuilder.Setup()"
17:17:20.493 [Error] 38060 <= "   at Avalonia.AppBuilder.SetupWithoutStarting()"
17:17:20.494 [Error] 38060 <= "   at Avalonia.DesignerSupport.Remote.RemoteDesignerEntryPoint.AppInitializer.ConfigureApp(IAvaloniaRemoteTransportConnection transport, CommandLineArgs args, Object obj)"
17:17:20.494 [Error] 38060 <= "   at Avalonia.DesignerSupport.Remote.RemoteDesignerEntryPoint.Main(String[] cmdline)"
17:17:20.494 [Error] 38060 <= "   at Avalonia.Designer.HostApp.Program.Main(String[] args)"
17:17:21.483 [Information]  Process exited
17:17:21.483 [Verbose]  Started PreviewerProcess.Stop()
17:17:21.485 [Information]  Stopping previewer process
17:17:21.485 [Verbose]  Finished PreviewerProcess.Stop()
17:17:21.483 [Error]  Connection error
```

### To Reproduce

1. Create a new Avalonia App, i'm using desktop and Android
2. Update the Avalonia to 11.2.0
Replace the content from the MainView.xaml with:

``` xaml
<StackPanel Orientation="Horizontal">
	<TimePicker />
</StackPanel>
```
Rebuild the project.

### Expected behavior

No designer errors

### Avalonia version

11.2.0

### OS

Windows, Android

### Additional context

I've created a repository [https://github.com/fredericoregateiro/AvaloniaUITimePicker](https://github.com/fredericoregateiro/AvaloniaUITimePicker)

**Comment by timunie** [comment]
You're using different Avalonia versions. Please tidy up this in your sample for Android and desktop. 

Or set it in the props file and reset the parameter for shared csproj file

**Comment by timunie** [comment]
Hopefully solved in 11.2.1 

It should have not been causing any breaking changes afaik. Sorry for the inconvenience.
